### ISSUE TITLE
Text following line-starting HTML tag gets split into new pargaraph

### ISSUE THREAD

**Issue by jessemortenson** [probablyup/markdown-to-jsx#220]
Been digging through the code and can't quite figure out the why of this behavior and how to change it. per https://probablyup.com/markdown-to-jsx/

```md
<ins>Insertion before text</ins> followed by some text

Some text <ins>insertion within</ins> text
```
becomes rendered as

```html
<ins>Insertion before text</ins>
<p> followed by some text</p>
<p>Some text <ins>insertion within</ins> text</p>
````

when I would instead expect:

```html
<p><ins>Insertion before text</ins> followed by some text</p>
<p>Some text <ins>insertion within</ins> text</p>
```

Any hint in the right direction? Thanks in advance.

**Comment by probablyup** [comment]
By default we expect html to be block level, so if it's the first thing on the line, it's left by itself and not wrapped in a paragraph. This could possibly be made smarter... I'll look into it.

**Comment by Buom01** [comment]
Any news ?

**Comment by probablyup** [comment]
Nope I’ve been extremely busy at work. I’ll probably have some free time
soon if someone else doesn’t fix it before then.

**Comment by Buom01** [comment]
I have done modifications, it works perfectly for my usage
https://github.com/Buom01/markdown-to-jsx/commit/435f3ba7f11d12b7b14d3245044c049e065e1271

**Comment by probablyup** [comment]
That breaks a lot of tests unfortunately :/

**Comment by cubian** [comment]
Any progress on this? We are facing the same issue.

**Comment by probablyup** [comment]
I’ve been very sick for the past month and am currently in the hospital so
you won’t hear from me for a bit

**Comment by Buom01** [comment]
> I’ve been very sick for the past month and am currently in the hospital so you won’t hear from me for a bit

@probablyup good recovery and health to you


> Any progress on this? We are facing the same issue.

Because I had time, I have updated my fork properly to make it working and make tests to pass.

I will make a pull request from https://github.com/buom01/markdown-to-jsx/ to https://github.com/probablyup/markdown-to-jsx/

**Comment by meglio** [comment]
Are you saying contributions will change the attitude and will somehow make more time available to the author? How much is needed?

**Comment by probablyup** [comment]
@meglio Absolutely, I have limited personal time and don't intend to spend my time on this project for anything other than vital security updates unless I can adequately compensate myself.

You're welcome to submit a PR if you want to spend your own time on it, or make a donation. As far as amount, I think at least $100 is fair; think about what you would charge for an hour or two of your own time.

**Comment by meglio** [comment]
Thanks for explaining @probablyup. Does your response also indicate that the project is not actively maintained anymore and we should not expect any significant updates / improvements / feature development?

If so, it is not an issue - I completely understand the motivation behind it; but could you mention that in the readme file as to set the correct expectations for those choosing a markdown library when starting a new project? That would be fair to state upfront. For instance, if I had read about the current state of the project (don't intend to spend my time on this project for anything other than vital security updates), I would have not chosen this library, and now it is extra time to migrate to another library that is actively maintained. So not saying that upfront penalizes library users with no reason. What do you think?

**Comment by probablyup** [comment]
Not at all, just that it will happen when I get the inspiration to do it if I'm not being compensated. You're welcome to choose whatever library you like; this one has a unique set of goals related to library size and extensibility that you might have a hard time replicating.

As always, you're welcome to contribute a fix yourself if you don't want to donate.

**Comment by xFEARx** [comment]
I think fixed in 7.1.6
